python/cudf/cudf/io/parquet.py

```diff
@@ -64,11 +64,23 @@ def read_parquet(
 
 
 @ioutils.doc_to_parquet()
-def to_parquet(df, path, *args, **kwargs):
+def to_parquet(
+    df,
+    path,
+    compression=None,
+    statistics="ROWGROUP",
+    engine="cudf",
+    index=None,
+    partition_cols=None,
```

Comment:
Need to error if someone passes an argument that isn't supported here for the time being.

Reply:
Unsure what you mean here? Argument as in an invalid parition_col value or do you mean something else?

Reply:
Since we don't actually support the index or partition_cols argument we need to error here if someone passes a non default value.

Reply:
i.e. if someone passes `['a', 'b']` to `partition_cols`, they'd expect our writer to properly partition based on that, but it doesn't quite yet.

Also, we should reorder these kwargs, it should be: `df, path, engine, compression, index, partition_cols, statistics` to match pandas behavior.

Reply:
Roger on the reordering. For the Error. Do we want to hard error and exit or utilize the "Warning"?

Reply:
We'd want to throw an exception if they pass a non-functional value to a parameter.